index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Product landing page</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+
+    <header id="header">
+      <nav id="nav-bar">
+          <img id="header-img" class="logo" src="/samsung.png" alt="samsung-logo">
+          <ul class="nav-menu">
+            <li class="nav-link">
+              <a href="#video" target="_blank">Samsung phone</a>
+            </li>
+            <li class="nav-link">
+              <a href="http://samsung." target="_blank">Samsung tablets</a>
+            </li>
+            <li class="nav-link">
+              <a href="http://samsung." target="_blank">Samsung house</a>
+            </li>
+          </ul>
+      </nav>
+    </header>
+
+    <div class="video">
+      <iframe itemid="video" width="420" height="315"
+         src="https://www.youtube.com/embed/sq4scK7hgxo" id="video">Samsung phone products
+      </iframe>
+    </div>
+    <section>
+        <form id="form">
+          <label> Email</label> <br>
+          <input
+          id="email"
+          type="email"
+          required
+          placeholder="username@beststartupever.com"
+          pattern=".+@beststartupever\.com"
+          title="Please provide only a Best Startup Ever corporate email address" />
+
+          <button id="submit"><a id="submit" href="https://www.freecodecamp.com/email-submit" target="_blank">submit</a></button>
+        </form>
```

Comment:
- [ ] Your comment is clear ✨ and states `Why` ❓, well done 👍 , however as suggested by the previous reviewer it would be better if you phrase your sentences as a suggestion instead of an order

> For example:
> 
> This is a polite suggestion: "I think it would be better to use semantic tags"
> 
> This reads as an order: "Use semantic tags"

So here in your comment instead of saying  "... don't overuse IDs as they are unique..."
**you can say** "I would suggest not to over use...", "It would be better if you don't over use..." or "I would recommend not using.."etc,